src/js/main.js

```diff
@@ -18,6 +18,12 @@ export function init(el, context, config) {
         const youTubeId = resp.sheets[config.sheetName][0].youTubeId;
         const chapters = resp.sheets[config.sheetChapter];
 
+        chapters.forEach( function(chapter, index){
```

Comment:
might be worth sorting chapters by `chapterTimestamp` ascending too (never trust external data 😜)